website/docs/docs/collaborate/explore-multiple-projects.md

```diff
@@ -4,31 +4,47 @@ sidebar_label: "Explore multiple projects"
 description: "Learn about project-level lineage in dbt Explorer and its uses."
 ---
 
-You can also view all the different projects and public models in the account, where the public models are defined, and how they are used to gain a better understanding about your cross-project resources.
+View all the projects and public models in your account (where public models are defined) and gain a better understanding of your cross-project resources and how they're used.
 
-The resource-level lineage graph for a given project displays the cross-project relationships in the DAG. The different icons indicate whether you’re looking at an upstream producer project (parent) or a downstream consumer project (child).
+The resource-level lineage graph for a project displays the cross-project relationships in the DAG, with an icon indicating whether  if it's a project resource. Notice the **PRJ** icon in the left side of the node name.
 
-When you view an upstream (parent) project, its public models display a counter icon in the upper right corner indicating how many downstream (child) projects depend on them. Selecting a model reveals the lineage indicating the projects dependent on that model. These counts include all projects listing the upstream one as a dependency in its `dependencies.yml`, even without a direct `{{ ref() }}`. Selecting a project node from a public model opens its detailed lineage graph, which is subject to your [permission](/docs/cloud/manage-access/enterprise-permissions).
+To view the project-level lineage graph, click the **View lineage** icon in the upper right corner from the main overview page:
+- This view displays all the projects in your account and their relationships.
+- Viewing an upstream (parent) project displays the downstream (child) projects that depend on it.
+- Selecting a model reveals its dependent projects in the lineage.
+- Click on an upstream (parent) project to view the other projects that reference it in the **Relationships** tab, showing the number of downstream (child) projects that depend on them. 
+  - This include all projects listing the upstream one as a dependency in its `dependencies.yml` file, even without a direct `{{ ref() }}`.
+  - Selecting a project node from a public model opens its detailed lineage graph, subject to your [permission](/docs/cloud/manage-access/enterprise-permissions).
 
-<Lightbox src="/img/docs/collaborate/dbt-explorer/cross-project-lineage-parent.png" width="70%" height="100" title="Cross-project lineage in a parent project"/>
+<Lightbox src="/img/docs/collaborate/dbt-explorer/cross-project-lineage-parent.png" width="100%" height="100" title="View your cross-project lineage in a parent project and the other projects that reference it by clicking the 'Relationships' tab."/>
```

Comment:
```suggestion
<Lightbox src="/img/docs/collaborate/dbt-explorer/cross-project-lineage-parent.png" width="100%" height="100" title="View your cross-project lineage in a parent project and the other projects that reference it by clicking the "Relationships" tab."/>
```

Reply:
the build is failing bc of the quotes. i think this is why i iniitally use apostrophes in the captions. going to go back to apostrophe's